src/openfhe.jl

```diff
@@ -381,3 +385,291 @@ function rotate(sv::SecureVector{<:OpenFHEBackend}, shift; wrap_by)
 
     secure_vector
 end
+
+
+############################################################################################
+# Matrix
+############################################################################################
+
+"""
+    init_matrix_rotation!(context::SecureContext{<:OpenFHEBackend}, private_key::PrivateKey,
+                          shifts, shape)
+
+Generate rotation keys for matrix rotation with `OpenFHE.EvalRotate` using the `private_key` and for the
+rotation indexes in `shifts`. `shifts` is a vector of pair values. The keys are stored in the given
+`context`. Positive shift defines rotation to the right/top, e.g. a rotation with a shift (1, 0):
+[1 2 3; 4 5 6; 7 8 9] -> [7 8 9; 1 2 3; 4 5 6].
+Negative shift defines rotation to the left, e.g. a rotation with a shift (0, 1):
+[1 2 3; 4 5 6; 7 8 9] -> [3 1 2; 6 4 5; 9 7 8].
```

Comment:
```suggestion
Generate rotation keys for matrix rotation with `OpenFHE.EvalRotate` using the `private_key` and for the
rotation indexes in `shifts`.
`shifts` is a pair of integers given as a tuple, or a list of tuples of integers.
The keys are stored in the given
`context`. A positive shift defines a rotation to the right/bottom, e.g., a rotation with shift `(1, 0)`:
[1 2 3; 4 5 6; 7 8 9] -> [7 8 9; 1 2 3; 4 5 6].
Negative shifts define rotation to the left/top, e.g., a rotation with a shift `(0, -1)`:
[1 2 3; 4 5 6; 7 8 9] -> [3 1 2; 6 4 5; 9 7 8].
```
Can you please re-check the shifts and their description in the examples? I tried to figure out the correct way using Julia's `circshift`, assuming that in the end we use the same nomenclature. But please make sure I didn't screw it up 😊

Reply:
Thank you! It is all correct.